lib/clis/wolkenkit/createDeployment/createDeploymentCommand.ts

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+import { buntstift } from 'buntstift';
+import { Command } from 'command-line-interface';
+import { createDeploymentManifests } from './createDeploymentManifests';
+import { CreateDeploymentOptions } from './CreateDeploymentOptions';
+import { exists } from '../../../common/utils/fs/exists';
+import fs from 'fs';
+import path from 'path';
+
+const createDeploymentCommand = function (): Command<CreateDeploymentOptions> {
+  return {
+    name: 'create-deployment',
+    description: 'Create deployment manifests for a wolkenkit application.',
+
+    optionDefinitions: [
+      {
+        name: 'deployment-directory',
+        description: 'the directory in which to create the deployment manifests',
+        type: 'string',
+        parameterName: 'path',
+        isRequired: false,
+        defaultValue: './deployment'
+      }
+    ],
+
+    async handle ({ options: {
+      verbose,
+      'deployment-directory': deploymentDirectory
+    }}): Promise<void> {
+      buntstift.configure(
+        buntstift.getConfiguration().
+          withVerboseMode(verbose)
+      );
+
+      try {
+        const appDirectory = process.cwd();
```

Comment:
```suggestion
        const applicationDirectory = process.cwd();
```

This needs to be changed in some additional places as well…